lib/hooks/update-action/index.js

```diff
@@ -39,7 +39,16 @@ module.exports = settings => {
         throw new BadRequestError(`User is not associated with ${transferToEstablishment.name}`);
       }
 
-      return model.patch({ action: 'transfer', data: { establishmentId: transferToEstablishmentId } });
+      const transferFromEstablishment = await Establishment.query().findById(project.establishmentId);
+
+      const meta = {
+        establishment: {
+          to: pick(transferToEstablishment, 'id', 'name'),
+          from: pick(transferFromEstablishment, 'id', 'name')
+        }
```

Comment:
This looks familiar :D

Reply:
hehe turns out it's a hell of a lot easier to attach the meta at the point of creating the transfer than trying to work it out depending on state in the UI :D